app/helpers/application_helper.rb

```diff
@@ -1,5 +1,11 @@
+include MarkdownContent
+
 module ApplicationHelper
   def theme
     League.current.theme || "default"
   end
+
+  def markdown(text)
+    MarkdownContent.markdown(text.to_s).html_safe
+  end
```

Comment:
Dunno if this was the right place to put this. Markdown support seems like a common enough thing to want at the application level, so I imported your module that dealt with Markdown and used it.

The `html_safe` call is lax, but I know that _for now_ the only place this is used is to render text sourced from the acheev's JSON so the resulting HTML will be fine.